outputs.tf

```
@@ -1,24 +1,14 @@
-output "arn" {
-  value       = aws_instance.example.arn
-  description = "The EC2 instance ARN."
+output "ssm_document" {
+  description = "The SSM document that can be used to create SSM SessionManager session in this account."
```

Grammar and spacing:
```suggestion
  description = "The SSM document that can be used to create SSM Session Manager sessions in this account."
```